x_and_o/main.c

```diff
@@ -0,0 +1,30 @@
+#include <stdio.h>
+#include <errno.h>
+#include "x_and_o.h"
+
+int main() {
+
+    char board[BOARD_SIZE][BOARD_SIZE];
+    initial_board(board);
+    print_board(board);
+    for (size_t i = 0; i < ((BOARD_SIZE * BOARD_SIZE)); i++)
+    {
+        char player = X_OR_O(i);
+        printf("Player turn - %c\n", player);
+        player_turn(board, player);
+        print_board(board);
+        printf("\n\n\n");
+        if (i >= (BOARD_SIZE * 2) - 3)
```

Comment:
A little ugly calculation.
You tried to make it generic with macro, but ended up with the number 3 there...